ios/Podfile.lock

```diff
@@ -1142,4 +1144,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c81fe8fa67af4bf6e27653958d177ac176ab0293
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
This shouldn't be changed. Am I right?
Be sure to run `bundle exec pod install`

Reply:
You're right! I added the correct Podfile.lock by executing `bundle exec pod install` check it out from this commit: 96241092b9489cf386061c234653c4ee93cc14f9